crates/cli/Cargo.toml

```diff
@@ -21,6 +21,7 @@ foundry-evm.workspace = true
 foundry-wallets.workspace = true
 
 foundry-compilers = { workspace = true, features = ["full"] }
+foundry-block-explorers.workspace = true
```

Comment:
I was stuck on whether or not to add this package dep. Makes more sense to than not to.